ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/altair/statetransition/epoch/EpochProcessorAltair.java

```diff
@@ -94,7 +94,6 @@ public void processParticipationUpdates(final MutableBeaconState genericState) {
     state.setPreviousEpochParticipation(state.getCurrentEpochParticipation());
 
     // Reset current epoch participation flags
-    state.getCurrentEpochParticipation().clear();
     state.getCurrentEpochParticipation().setAll(SszByte.ZERO, state.getValidators().size());
```

Comment:
not related nit: TBH I'd remove the `SszMutableCollection.setAll(SszElementT, int)` overload at all. Looks a bit misleading